### ISSUE TITLE
Create structure for the theme sprint meeting

### ISSUE THREAD

**Issue by menor** [resmio/git-along#22]
More on this later

**Comment by fabiennepimenta** [comment]
Here's a summary of what Theme sprints are with the different stages of "Sketch/Design" "Planning meeting" and "Validation" 

 <img width="796" alt="screen shot 2016-09-20 at 17 16 41" src="https://cloud.githubusercontent.com/assets/3285116/18676493/1378f804-7f56-11e6-9390-909e4266bdd4.png">

_from LeanUX_

**Comment by menor** [comment]
What are your feelings about testing every week? Don't you think it will be too much time consuming? What about once every sprint?

**Comment by fabiennepimenta** [comment]
I agree that testing every week can be "exhausting"  – for us and for the testers. We should also test what makes sense to test and not just because it's scheduled ( I guess ). So yes, I agree that once every sprint sounds good.
________________________________________________________________________________________________________

</br>
Something that we can also do ( that I really like, because it give us a bit more responsibility about a project) is to create teams. Webapp team, Widget team, API team, iPad team, etc.  One example: Michael will be the project Manager for all teams, I could be the Product Design for Webapp, Widget and iPad, Jose & Steven on the Webapp and Widget, etc... 

And I say this because I believe that if you are inside a team, you will be more in to support it, ask more times what is going one, more "small meetings" with team members about those project, etc.
And people outside the product team will also know the teams that we have inside so it's easier for them to know to who they can make questions.
 
But of course, everyone need to agree with this and be comfortable with this kind of "structure".  
________________________________________________________________________________________________________

</br>
About Design & Dev sync I would say that I always need to be 1 sprint in front of you – but with discussion between.

**Comment by menor** [comment]
I like the team idea for bigger companies, but maybe here everybody will end up on every team except for the iPad app.
